### DISCUSSION TITLE
Not Synced after a forced restart

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2237] [Q&A]
Started too many plots in parallel and it bogged down the machine. Had to restart. Then it stayed "Not Synced" for ever. Anyway to fix this?  Find the following error in the shell,

Last login: Wed Apr 21 07:19:33 on ttys000
/Applications/Chia.app/Contents/MacOS/Chia ; exit;                              
USER@iMac ~ % /Applications/Chia.app/Contents/MacOS/Chia ; exit;
/Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon
true
Error loading config - using defaults
startUrl file:///Applications/Chia.app/Contents/Resources/app.asar/build/renderer/index.html
/Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon
true
/Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon
true
Running python executable: 
child process success
(node:2946) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
No cert
Chia directory /Users/owner27/.chia/mainnet
/Users/USER/.chia/mainnet already exists, no migration action taken
No cert
Have cert

**Comment by VaulB** [comment]
I got a similar problem and this is not the first time. Chia GUI was white after being on for some time. Reload and force reload are always useless. When I booted, then it wouldn't sync anymore. This is unacceptable. Syncing takes so long time. I have already lost couple of days worth plotting and farming because of this. Getting more and more frustrated every day. The only thing left is to remove .chia folde "again" and start syncing all over again. I will try to make a backup copy. Hopefully that will save me next time...
